manager/assets/modext/widgets/core/modx.searchbar.js

```diff
@@ -97,45 +110,52 @@ MODx.SearchBar = function(config) {
                     this.tpl.type = null;
                 }
             }
-            ,focus: this.focusBar
-            ,blur: this.blurBar
+            ,focus: {
+                fn: function(){
+                    this.focusBar();
```

Comment:
If I did that, it was probably to add the scope to it - but doesn't look like that's the case as the scope is defined one element up. Not sure.